packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -83,8 +83,9 @@ class OutlinedButton extends ButtonStyleButton {
     super.autofocus = false,
     super.clipBehavior,
     super.statesController,
+    Widget? icon,
```

Comment:
looking at the code this `_icon` is just to check whether a padding is needed. The developer are still expect to build an icon in the child by themselves.

might as well let this be `bool addPaddding`.